tests/cypress/e2e/pickers/externalpicker.cy.ts

```diff
@@ -0,0 +1,38 @@
+import {contentTypes} from '../../fixtures/contentEditor/pickers/contentTypes';
+import {assertUtils} from '../../utils/assertUtils';
+import {AccordionItem} from '../../page-object/accordionItem';
+import {JContent} from '../../page-object/jcontent';
+import {createSite, deleteSite, Dropdown, enableModule, getComponentByRole} from '@jahia/cypress';
+
+describe('Picker tests', () => {
+    let jcontent: JContent;
+
+    before(function () {
+        createSite('externalPickerTest', {
+            templateSet: 'dx-base-demo-templates',
+            serverName: 'localhost',
+            locale: 'en'
+        });
+        enableModule('jcontent-test-module', 'externalPickerTest');
+        enableModule('qa-module', 'externalPickerTest');
+        cy.loginAndStoreSession(); // Edit in chief
+        jcontent = JContent.visit('externalPickerTest', 'en', 'content-folders/contents');
+    });
+
+    after(function () {
+        deleteSite('externalPickerTest');
+    });
+
+    // Tests
+
+    it('should display content reference picker', () => {
+        const pickerField = jcontent
+            .createContent('Pickers')
+            .getPickerField('qant:pickers_filepicker');
+        const picker = pickerField.open();
+        getComponentByRole(Dropdown, 'picker-selector').select('Custom picker');
+        cy.contains('Test picker');
+        cy.get('button[data-sel-role="custom-selector"]').click();
+        pickerField.assertValue('user.jpg');
+    });
```

Comment:
We could add a test on a rich text ensuring that RichText is correctly handling full URL when selecting an image from the test DAM ?

I know we can't test references as it would imply to implement an DAM EDP along the picker, but rich text seem's testable, what do you think ?